File: 151001/Novik/DC/src/Discussion/Models/Entity/Post.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Discussion.Models.Entity;

public class Post
{

    public long id { get; set; }
    
    public long? newsId { get; set; }
    
    public String? content { get; set; }
    
    public String country { get; set; }
    
}

File: 151001/Baran/Lab3/src/main/java/by/bsuir/entities/Post.java
package by.bsuir.entities;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "tbl_post", schema = "distcomp")
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;
    @ManyToOne (cascade=CascadeType.REFRESH)
    @JoinColumn (name="issue_id")
    private Issue issue;
    @Column(name = "content")
    private String content;
}
